### PR TITLE
[ duktape] Update hash for pip.

### PR DESCRIPTION
**Pull request by LilyWangL**
1. Update hash for `pip`.

Related issue: #7528

### CODE REVIEW

**Comment by wangli28** [comment]
Duktape build failed on Linux because it requires tools from the system package manager: `python-yaml`. This tool needs to install manually. There are comments in duktape's portfile.cmake.
```
if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "Linux")
    message("${PORT} currently requires the following tools from the system package manager:\n    python-yaml\n\nThis can be installed on Ubuntu systems via apt-get install python-yaml PYTHON2-yaml (depending on your current python default interpreter)")
endif()
```

**Comment by Voskrese** [comment]
there will be a constant error when changing the file
can use versioned?

https://bootstrap.pypa.io/3.3/
https://bootstrap.pypa.io/3.3/get-pip.py

**Comment by dan-shaw** [comment]
@wangli28 Could you change the URL in `vcpkg_download_distfile` so that it gets a specific version of pip instead. That way, if `pip` were to update we wouldn't have to change the `SHA512` again.

Some pip versions here: https://github.com/pypa/get-pip

File: ports/open62541/CONTROL [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Source: open62541
-Version: 0.3.0-3
+Version: 0.3.0-4
+Homepage: https://github.com/open62541/open62541
 Description: open62541 is an open source C (C99) implementation of OPC UA licensed under the Mozilla Public License v2.0.
```
Comment (dan-shaw) [inline ports/open62541/CONTROL]: This appears to have been fixed in #7607, so I don't think we need this change anymore.

File: ports/duktape/portfile.cmake [hunk 2]
```diff
@@ -28,9 +28,9 @@ vcpkg_add_to_path("${PYTHON2_DIR}")
 if(NOT EXISTS ${PYTHON2_DIR}/easy_install${EXECUTABLE_SUFFIX})
     if(NOT EXISTS ${PYTHON2_DIR}/Scripts/pip${EXECUTABLE_SUFFIX})
         vcpkg_download_distfile(GET_PIP
-            URLS "https://bootstrap.pypa.io/get-pip.py"
+            URLS "https://bootstrap.pypa.io/3.3/get-pip.py"
```
Comment (dan-shaw) [inline ports/duktape/portfile.cmake]: We are using `python2`, so we should use an earlier version of `pip`.

Comment (wangli28) [reply ports/duktape/portfile.cmake]: There are multiple versions of the file. Shouldn't we use the latest version?

**Pull request merged**